File: Contract.Architecture.Frontends/Contract.Architecture.Web.Core/src/app/model/gegoennter-kundenstamm/gegoennte-kunden/dtos/i-gegoennter-kunde.ts
import { ApiGegoennterKunde } from './api/api-gegoennter-kunde';

export interface IGegoennterKunde {
    id: string;
    name: string;
    gegoennterName: string;
    gegoennterBoolean?: boolean;
    gegoennterDateTime?: Date;
    gegoennterDouble?: number;
    gegoennterGuid?: string;
    gegoennterInteger?: number;
    besteBankId: string;
}

export class GegoennterKunde {
    public static fromApiGegoennterKunde(apiGegoennterKunde: ApiGegoennterKunde): IGegoennterKunde {
        return {
            id: apiGegoennterKunde.id,
            name: apiGegoennterKunde.name,
            gegoennterName: apiGegoennterKunde.gegoennterName,
            gegoennterBoolean: apiGegoennterKunde.gegoennterBoolean,
            gegoennterDateTime: apiGegoennterKunde.gegoennterDateTime,
            gegoennterDouble: apiGegoennterKunde.gegoennterDouble,
            gegoennterGuid: apiGegoennterKunde.gegoennterGuid,
            gegoennterInteger: apiGegoennterKunde.gegoennterInteger,
            besteBankId: apiGegoennterKunde.besteBankId,
        };
    }
}

File: Contract.Architecture.Backends/Contract.Architecture.Backend.Core/Contract/Logic/Modules/GegoennterKundenstamm/GegoennteKunden/DTOs/IGegoennterKunde.cs
using System;

namespace Contract.Architecture.Backend.Core.Contract.Logic.Modules.GegoennterKundenstamm.GegoennteKunden
{
    public interface IGegoennterKunde
    {
        Guid Id { get; set; }

        string Name { get; set; }

        string GegoennterName { get; set; }

        bool? GegoennterBoolean { get; set; }

        DateTime? GegoennterDateTime { get; set; }

        double? GegoennterDouble { get; set; }

        Guid? GegoennterGuid { get; set; }

        int? GegoennterInteger { get; set; }

        Guid BesteBankId { get; set; }
    }
}